### PR TITLE
PMM-0000 export repo url in .git-sources

### PR DESCRIPTION
**Pull request by ritbl**

### CODE REVIEW

File: ci.py [hunk 1]
```diff
@@ -190,7 +191,8 @@ def check_deps(self):
             repo_path = '/'.join(target_url.split('/')[-2:])
             target_branch = dep['branch']
             repo = github_api.get_repo(repo_path)
-            head = f'{repo.organization.name}:{target_branch}'
+            org = repo.organization.name if repo.organization else repo.owner.login
+            head = f'{org}:{target_branch}'
```
Comment (ritbl) [inline ci.py:195]: to support non-org repos

Comment (ritbl) [reply ci.py:195]: [here is an example](https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-submodules/detail/PR-2220/2/pipeline#step-17-log-63) where it fails

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2221-8cae5dd
client docker - perconalab/pmm-client-fb:PR-2221-8cae5dd
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2221-8cae5dd.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2221-8cae5dd&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2221-8cae5dd.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2221-8716d79
client docker - perconalab/pmm-client-fb:PR-2221-8716d79
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2221-8716d79.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2221-8716d79&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2221-8716d79.tar.gz

**Review by nikita-b — APPROVED** [review]

**Review by yurkovychv — APPROVED** [review]
Thanks a lot @ritbl

**Comment by JNKPercona** [comment]
API tests have failed, Please check: API: https://pmm.cd.percona.com/job/pmm2-api-tests/8837/

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/8296/

**Comment by JNKPercona** [comment]
pmm2-client testsuite has failed, Please check: BATS: https://pmm.cd.percona.com/job/pmm2-testsuite/8297/

**Pull request merged**